src/pages/explorer/spaceApi/SpaceApi.jsx

```diff
@@ -196,12 +196,8 @@ const mapStateToProps = (state, ownProps) => {
       'uiState.explorer.spaceApi.query',
       templates.defaultTemplate
     ),
-    useInternalToken: get(
-      state,
-      'uiState.explorer.useInternalToken',
-      true
-    ),
-    userToken: get(state, 'uiState.explorer.userToken'),
+    useInternalToken: getLastUsedValues(projectId).useInternalToken,
```

Comment:
We should be first taking these values from the redux. If not available in redux only then should we take these values from the localStorage